aws.py

```diff
@@ -122,7 +122,7 @@ def cf_bucket():
     return (None, None)
 
 
-def fetch_bucket(bucket_name=None, clear_tables=True):
+def get_bucket(bucket_name=None):
```

Comment:
Having both a `get_bucket` and a `fetch_bucket` function in the module seems confusing.  What would you think of adding a comment line to each to explain, and maybe renaming `get_bucket` to `_get_bucket`?